bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -116,17 +120,26 @@
           <div class="video" data-video-id="R_qDaLQ8ghg"></div>
         </div>
       </li>
+      <li class="feature">
```

Comment:
We're repeating the same `Page Inspector` block twice in the template here. Could we avoid the repetition by only wrapping the l10n tag around the first item which changes, instead of repeating this here?